extensions/amp-user-notification/0.1/amp-user-notification.js

```diff
@@ -126,6 +125,11 @@ export class AmpUserNotification extends AMP.BaseElement {
       assertHttpsUrl(this.dismissHref_, this.element);
     }
 
+    const alwaysShowAttr = this.element.getAttribute('always-show');
+    /** @private @const {boolean} */
+    this.alwaysShow_ = (this.element.hasAttribute('always-show') &&
+        alwaysShowAttr != '0' && alwaysShowAttr != 'false');
```

Comment:
The problem with no value is cases like `always-show="false"` or `always-show="true"` what each of these mean? Do they error?